main.go

```diff
@@ -1,29 +1,24 @@
 package main
 
 import (
-	"fmt"
-	"github.com/vkhichar/asset-management/config"
-	"github.com/vkhichar/asset-management/handler"
-	"github.com/vkhichar/asset-management/repository"
-	"net/http"
+	"os"
+
+	"github.com/vkhichar/asset-management/server"
 )
 
 func main() {
-	err := config.Init()
```

Comment:
this should gracefully handle the error.
if there are no arguments passed. this will through the index out of range error